amr-wind/wind_energy/ABLStats.cpp

```diff
@@ -228,7 +228,9 @@ void ABLStats::compute_zi(const h1_dir& h1Sel, const h2_dir& h2Sel)
         for (size_t i = 0; i < m_ncells_h1 * m_ncells_h2; i++) {
             m_zi += gtemp_grad[i].max_grad_loc;
         }
-        m_zi /= (m_ncells_h1 * m_ncells_h2);
+        m_zi /=
+            (static_cast<double>(m_ncells_h1) *
+             static_cast<double>(m_ncells_h2));
```

Comment:
amrex::Real?

Reply:
I looked at that but `m_zi` is declared a `double`.

Reply:
ohhh because of netcdf. well double should be fine. Or we can cast the double to an amrex::Real at the netcdf call?

Reply:
That would probably be better, but there are a lot of explicit `double` around so probably would tackle those later if possible.

Reply:
@michaeljbrazell For things that write out to NetCDF interface, I think that we should be explicit about `float/double` instead of using `amrex::Real` This will be necessary to catch issues if an external AMReX was compiled with single precision. For often used NetCDF methods it might be useful to provide a templated version that can be specialized on `amrex::Real` at the call site.